keras_retinanet/bin/debug.py

```diff
@@ -181,6 +182,7 @@ def csv_list(string):
     parser.add_argument('--random-transform', help='Randomly transform image and annotations.', action='store_true')
     parser.add_argument('--image-min-side', help='Rescale the image so the smallest side is min_side.', type=int, default=800)
     parser.add_argument('--image-max-side', help='Rescale the image if the largest side is larger than max_side.', type=int, default=1333)
+    parser.add_argument('--image-extension',   help='Declares the dataset images\' extension.', default='.jpg')
```

Comment:
This argument is only necessary for Pascal VOC, so we should add it to that subparser.

Reply:
In all `bin` script files, right?